packages/expo-file-system/src/FileSystem.ts

```diff
@@ -69,6 +69,15 @@ export async function readAsStringAsync(
   return await ExponentFileSystem.readAsStringAsync(fileUri, options || {});
 }
 
+export async function contentUriFromFile(
```

Comment:
could we call this function `getContentUriAsync` instead? our convention is to add the `Async` suffix to methods which return a promise, and this name is a little more concise.